src/core/Akka.Streams.Tests/Dsl/LastElementSpec.cs

```diff
@@ -56,7 +56,7 @@ public void A_stream_via_LastElement_should_materialize_to_materialize_to_None_f
         [Fact]
         public void A_stream_via_LastElement_should_materialize_to_the_last_element_emitted_by_a_source_before_it_failed()
         {
-            var t = Source.Unfold(1, n => n >= 3 ? throw new Exception() : Tuple.Create(n + 1, n + 1))
+            var t = Source.UnfoldInfinite(1, n => n >= 3 ? throw new Exception() : Tuple.Create(n + 1, n + 1))
```

Comment:
This is one way to fix compilation error after adding `Option` requirement for `Unfold` method. Also, this allows to verify that `UnfoldInfinite` works well.